src/main/java/com/soapboxrace/core/api/AddFriendRequest.java

```diff
@@ -29,6 +37,16 @@
     public Response addFriendRequest(@HeaderParam("securityToken") String securityToken,
                                      @QueryParam("displayName") String displayName) {
         Long activePersonaId = tokenSessionBO.getActivePersonaId(securityToken);
+
+        if (tokenSessionBO.isAdmin(securityToken) && displayName.startsWith("/")) {
+            String[] abuseCase = displayName.split(" ");
+            Long abuserId = personaDAO.findByName(abuseCase[1].trim()).getPersonaId();
+            if (abuserId != null) {
```

Comment:
this will not work the way you think it will
```java
PersonaEntity abuserPersona = personaDAO.findByName(abuseCase[1].trim());
if (abuserPersona != null) {
    adminBO.sendCommand(tokenSessionBO.getActivePersonaId(securityToken), abuserPersona.getPersonaId(), displayName);
}
```

Reply:
Gotta try that, well good thing the code did work fine when i tested it (ban, unban, kick) and it didn't require the driver to be online :+1: